my-dsl.md

```diff
@@ -2,37 +2,46 @@
 
 # Internal or external?
 _Is the language implemented as an internal or external DSL? 
 Justify your answer._
 
+The language is implemented as an external DSL, since it is intended to be a supplement to other programs. It can be used in other applications as a replacement for booleans. FCL has its own syntax separate from the application in which it is being used. Since the entire purpose of the language is to implement fuzzy logic, FCL has its own unique features and is not merely a subset of features of another language, which would be characteristic of an internal DSL. 
 
 # Host language
 _What language(s) was (were) used to implement the DSL?_
 
+FCL is a set of standards defined in the IEC 61131-7 standard for programmable controllers. Compilers and parsers for FCL have been implemented in several different languages. The language itself is constrained by the aforementioned standards. One example of an FCL library is jFuzzyLogic, which is an implementation of FCL in Java.
 
 # Benefits
 _Identify one potential benefit of the DSL: how is a programmer's life or a 
 company's bottom line made easier by the existence of this language?_
 
+Using this DSL, programmers can easily make use of fuzzy logic. With FCL, the programmer can define a set of general rules to categorize fuzzy values (unknowns ranging from 0 to 1) in terms of the specific problem they are trying to solve. The language allows the user to add their own sets of objects and categories in order to solve fuzzy logic problems within their more specific domain. This customizability is very convenient when working in the context of a real-world problem in which the answer is not a simple true or false value.
 
 # Drawbacks
 _Identify one potential drawback of the DSL: what does a programmer or company 
 lose by using this DSL instead of a general-purpose language?_
+
+One potential drawback of the language is that it does not include many keywords. Though FCL is very user-friendly (statements begin to look more like sentences than code snippets), some inuitive modifiers do not exist in the language. For example, the language supports `IF` and `THEN` but does not include a variable modifier such as `VERY`, which would place the object being modified into a different category than if the modifier was not present. As mentioned before, the language does not support features of a more general-purpose language, such as loops. A user loses these features by choosing to use this particular DSL. 
```

Comment:
If FCL is being implemented in another language, do users have access to loops from that host language?